Source/Scene/Model.js

```diff
@@ -1186,8 +1187,21 @@ define([
         var cacheKey = defaultValue(options.cacheKey, getAbsoluteUri(url));
 
         options = clone(options);
-        options.basePath = getBaseUri(url, true);
-        options.cacheKey = cacheKey;
+        var providedBasePath = options.basePath;
+        if (!defined(options.basePath)) {
+            options.basePath = getBaseUri(url, true);
+        }
+        if (!defined(options.cacheKey)) {
+            if (defined(providedBasePath)) {
+                options.cacheKey = cacheKey + options.basePath;
```

Comment:
It is ok this way since the cache key doesn't need to be a url, it just needs to be a unique way to identify a model.